maestro-cli/src/main/java/maestro/cli/session/MaestroSessionManager.kt

```diff
@@ -89,6 +90,7 @@ object MaestroSessionManager {
             SessionStore.withExclusiveLock {
                 heartbeatFuture.cancel(true)
                 SessionStore.delete(sessionId, selectedDevice.platform)
+                ScreenReporter.reportMaxDepth()
```

Comment:
Should we place this call under `runCatching { }`?  Just to avoid spilling any errors during the shutdown process

Reply:
Makes sense done 💯